app/Domains/Users/Policies/UserPolicy.php

```diff
@@ -25,7 +25,7 @@ public function delete(User $user): bool
     /**
      * Determine whether the authenticated user can view other users or not.
      *
-     * @param  User $user The resource entity from the authenticated user.
+     * @param User $user The resource entity from the authenticated user.
```

Comment:
Hey Can u align the User class with the same start as the return type in the docblock.